### PR TITLE
chore: add certified columns to top of list

### PR DESCRIPTION
**Pull request by pkdotson**
### SUMMARY
This pr add certified columns to top of datasourcepanel list. This pr is follow up 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

before

https://user-images.githubusercontent.com/17326228/135363783-13a544ba-324b-47c2-82a0-a76000f924ec.mov

after

https://user-images.githubusercontent.com/17326228/135363680-6a76da36-861a-4a1f-b62a-fbcef187d39b.mov

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/explore/components/DatasourcePanel/index.tsx [hunk 1]
```diff
@@ -253,6 +257,12 @@ export default function DataSourcePanel({
     setInputValue('');
   }, [columns, datasource, metrics]);
 
+  const sortCertifiedFirst = (slice: ColumnMeta[]) => {
```
Comment (geido) [inline superset-frontend/src/explore/components/DatasourcePanel/index.tsx:260]: I think here for brevity you could just sort the slice array, something like:

`slice.sort((a, b) => (b. is_certified - a. is_certified));`

**Review by geido — APPROVED** [review]
LGTM with a nit! Thank you

File: superset-frontend/src/explore/components/DatasourcePanel/index.tsx (hunk 1 above)
Comment (michael-s-molina) [reply superset-frontend/src/explore/components/DatasourcePanel/index.tsx:260]: Agreed

Comment (villebro) [reply superset-frontend/src/explore/components/DatasourcePanel/index.tsx:260]: Does this retain the order within the certified vs non-certified groups? Doing it like it's done here should at least ensure the order is retained.

Comment (geido) [reply superset-frontend/src/explore/components/DatasourcePanel/index.tsx:260]: @villebro I think the order is retained. Here is a short demo https://jsfiddle.net/9c0fjpwv/

Comment (pkdotson) [reply superset-frontend/src/explore/components/DatasourcePanel/index.tsx:260]: Yes this works thanks for the suggesiton.

File: superset-frontend/src/explore/components/controls/FilterControl/AdhocFilter/index.js [hunk 2]
```diff
@@ -154,19 +154,24 @@ export default class AdhocFilter {
       op => OPERATOR_ENUM_TO_OPERATOR_TYPE[op].operation,
     );
     if (this.expressionType === EXPRESSION_TYPES.SIMPLE) {
+      // console.log('isValid Meth', this.expressionType)
       if (nullCheckOperators.indexOf(this.operator) >= 0) {
+        // console.log('Null Op --> opertator', this.operator)
+        // console.log('subject', this.subject)
```
Comment (michael-s-molina) [inline superset-frontend/src/explore/components/controls/FilterControl/AdhocFilter/index.js:160]:
```suggestion
```

File: superset-frontend/src/explore/components/controls/FilterControl/AdhocFilter/index.js [hunk 3]
```diff
@@ -154,19 +154,24 @@ export default class AdhocFilter {
...
         return !!(this.operator && this.subject);
       }
       if (truthCheckOperators.indexOf(this.operator) >= 0) {
+        // console.log('i hit in truthCheckers', this.subject)
```
Comment (michael-s-molina) [inline superset-frontend/src/explore/components/controls/FilterControl/AdhocFilter/index.js:164]:
```suggestion
```

File: superset-frontend/src/explore/components/controls/FilterControl/AdhocFilter/index.js [hunk 4]
```diff
@@ -154,19 +154,24 @@ export default class AdhocFilter {
...
         return !!(this.subject && this.comparator !== null);
       }
       if (this.operator && this.subject && this.clause) {
+        // console.log('i trio check', this.operator, 'subject ->', this.subject, 'clause', this.clause)
```
Comment (michael-s-molina) [inline superset-frontend/src/explore/components/controls/FilterControl/AdhocFilter/index.js:168]:
```suggestion
```

File: superset-frontend/src/explore/components/DatasourcePanel/index.tsx [hunk 5]
```diff
@@ -253,6 +257,11 @@ export default function DataSourcePanel({
...
+    const sorted = slice.sort((a, b) => b.is_certified - a.is_certified);
+    return sorted;
+  };
```
Comment (michael-s-molina) [inline superset-frontend/src/explore/components/DatasourcePanel/index.tsx:263]:
```suggestion
  const sortCertifiedFirst = (slice: ColumnMeta[]) =>
    slice.sort((a, b) => b.is_certified - a.is_certified);
```

**Review by michael-s-molina — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**